server/deployment/treetracker-admin-api-deployment.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: treetracker-admin-api
+  labels:
+    app: treetracker-admin-api
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: treetracker-admin-api
+  template:
+    metadata:
+      labels:
+        app: treetracker-admin-api
+    spec:
+      containers:
+      - name: treetracker-admin-api
+        image: greenstand/treetracker-admin-api:1.0.0-latest
+        ports:
+        - containerPort: 80
+        env:
+          - name: JWT_SECRET
+            valueFrom:
```

Comment:
So we need to create these secrets per cluster, yes? Wonder if we should set contents using env vars and template using helm

Reply:
Yes, these secrets are pre-requisites for each cluster. So whenever a new namespace is built out, these secrets need to be present prior to deploying any apps. So whoever is the cluster administrator, should also be responsible for deploying these secrets at the same time. 

I agree with helm. We need some sort of templating for the long-term solution.